### ISSUE TITLE
Install a custom library with data_files in pipenv fails

### ISSUE THREAD

**Issue by fernando24164** [pypa/pipenv#4028]
### Install a custom library in pipenv fails

I have a setup.py file with data_files for config files.
When I run `pipenv run python setup.py install` it generates a egg file in
virtual environment but without the data files set in setup.py

### Expected result

Create a egg folder with data_files setting in setup.py file

### Actual result

An egg file in site-packages folder without the data files setting in setup.py

**Comment by nicktimko** [comment]
If you create a virtual environment, do you have the same issue? The problem may be interactions between pip/setuptools rather than anything Pipenv.  

```
python -m venv testvenv
testvenv/bin/python setup.py install
cd testvenv/lib/python?.?/site-packages/  # are they in here?
```

Other things to check:
1. Build an `sdist` and see if the data files are packaged in the `tar.gz`
2. Move to a `MANIFEST.in` approach. Example from a package I have:
    * https://github.com/nicktimko/claptrap/blob/6ab439f/MANIFEST.in
    * https://github.com/nicktimko/claptrap/blob/6ab439f/setup.py#L21

**Comment by fernando24164** [comment]
Thanks for the feedback, yes I tested in another virtualenv and it works fine.
I also tested with sdist command and with a properly MANIFEST file everything is ok.

The problem is when I do `pipenv run python setup.py install` and then check the virtualenv managed by pipenv. In site-packages there is not `my_lib.egg` as a folder with all my config files, instead it is a file.

That's why I created this issue everything seems to work until I try to install with pipenv,  just in case would be a strange bug in pipenv.

**Comment by nicktimko** [comment]
Sounds good then. I'd try to create a minimal, complete example that you could point to though. Your report doesn't have any way to recreate the issue as there is no `setup.py` pointed to.

**Comment by fernando24164** [comment]
My problem is with a custom library which use cython. I will try to create a minimal to try to replicate.
Sorry for not sharing anything, I check this 'possible bug' in my company

**Comment by matteius** [comment]
This should be resolved on `pipenv==2022.8.19`
